plugwise/helper.py

```diff
@@ -698,6 +710,23 @@ def heating_valves(self):
 
         return None if loc_found == 0 else open_valve_count
 
+    @staticmethod
+    def energy_diff(measurement, net_string, f_val, direct_data):
```

Comment:
If you move this one outside of the class, to the top of the file, there's no need to use `@static_method` and you can call it by using the function-name, no longer needed to call it via `self`.

Reply:
if we would use it across (multiple) classes (from that same include) that would make sense, so if we envision the same being used by USB that indeed would be something to move

Reply:
I was just being practical, I've moved them outside the class so they can be moved into `util.py` at a later moment, just like the other functions that are already in that file :)

Reply:
I'll move it out next commit